spec/enumerable_spec.rb

```diff
@@ -0,0 +1,137 @@
+require './lib/enumerable_method'
+# rubocop:disable Layout/LineLength
+
+describe Enumerable do
+  describe '#my_each' do
+    it 'should return each element in an array' do
+      expect([2, 4, 6].my_each { |num| puts num }).to eq([2, 4, 6])
+    end
+  end
```

Comment:
For each of the tests, you should test the positive and negative ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)). For example:
```ruby
describe '#my_each' do
    it 'returns each element in an array' do
      expect([2, 4, 6].my_each { |num| puts num }).to eq([2, 4, 6])
    end

    it 'does not return an element that is absent in an array' do
      expect([2, 4, 6].my_each { |num| puts num }).not_to eq([2, 4, 9])
    end
end
```

As illustrated above, we tested for what it's supposed to be, then tested for what it's not supposed to be. That way our tests can be more encompassing and can't be easily broken.